src/view/adminhtml/web/js/datepicker.js

```diff
@@ -0,0 +1,22 @@
+//This script fixes incorrect positioning in modal datepickers.
+//This file should be removed after magento fixes the datepickers.
+
+require(["jquery"], function ($) {
+    $(window).on("load", function () {
+        if (window.location.pathname.includes("/admin/catalog/product/edit/id")) {
+            (($) => {
```

Comment:
You already have `$` defined on line 4.
And the code is encapsulated in the function scope.
So there is no need to wrap your code to the self-invoking function.
Please remove it.